src/timetable.cc

```diff
@@ -23,6 +23,21 @@ void timetable::resolve() {
   }
 }
 
+std::string_view timetable::route_short_name(const transport_idx_t t) const {
+  auto const trip_idx =
+      merged_trips_[transport_to_trip_section_[t].front()].front();
```

Comment:
one transport can be N concatenated trips via `block_id`
so for debugging purposes, it's sometimes okay to only return the first
but for the API we want the correct route/trip name to be displayed (on the map, etc)